### ISSUE TITLE
🐛 Section A: researcher profile URL is not registering as a required field

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/dac-ui#285]
## Describe the bug
@anncatton Justin needs to do some investigation to see if it's related to the API, so we may need your help on this one. 

- [ ] Section A: Researcher Profile URL is required but doesn't show the error. Have a look at this field, as when I fill it out and the section is complete, then I erase the Researcher URL, the section stays complete
![image](https://user-images.githubusercontent.com/18424138/126805050-ae52323a-a91b-49f0-a9b5-4dbb20857820.png)
![image](https://user-images.githubusercontent.com/18424138/127015949-aa977fed-1aa9-437a-a04e-fe3ee3df59cf.png)

## Steps To Reproduce

Steps to reproduce the behaviour:
1. Start a new application and go to section A
2. fill out first name, all validation appears except the researcher profile URL.
3. fill out all of section A including researcher profile url, the section is checked off
4. erase researcher url, you see the form field validation error but that section A is still checked off as complete

## Expected behaviour
Researcher Profile URL should register as a required field.

**Comment by caravinci** [comment]
Upon creating a new application in Dev (to ensure I have the latest API version + schema available), I've added a url, which I verified was persisted correctly.

Then I proceeded to remove it, and this was the response:

![image](https://user-images.githubusercontent.com/2107110/127062452-2964a0d6-c533-40f5-a37a-f81e1a5b812f.png)


Of note, unlike those other fields, website is not listed in the `errorsList`; indicating it is not being tracked as a required field. UI validation does show and error message accordingly, so no changes are needed in the Front End at this time.
